File: src/main/java/org/jukeboxmc/item/enchantment/EnchantmentEfficiency.java
package org.jukeboxmc.item.enchantment;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class EnchantmentEfficiency extends Enchantment {

    @Override
    public short getId() {
        return 15;
    }

    @Override
    public int getMaxLevel() {
        return 5;
    }

    @Override
    public int getWeight() {
        return 10;
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/enchantment/EnchantmentEfficiency.kt
package org.jukeboxmc.server.item.enchantment

import org.jukeboxmc.api.item.enchantment.Rarity

class EnchantmentEfficiency : JukeboxEnchantment() {

    override fun getId(): Int {
        return 15
    }

    override fun getMaxLevel(): Int {
        return 5
    }

    override fun getMinCost(level: Int): Int {
        return 1 + (level - 1) * 10
    }

    override fun getMaxCost(level: Int): Int {
        return super.getMaxCost(level) + 50
    }

    override fun getRarity(): Rarity {
        return Rarity.COMMON
    }

}
